solver/pU/pEqn.H

```diff
@@ -199,9 +199,9 @@ License
             (
                 phiHbyA.boundaryField()
               - alphaf.boundaryField()
-               *(mesh.Sf().boundaryField() & Ua.boundaryField())
+               *((mesh.Sf().boundaryField() & Ua.boundaryField())
```

Comment:
Are you sure?
I think there's a bug with the brackets...
Can you check?

Reply:
re-checked and re-corrected!